packages/directory-plugin/example/public/index.js

```diff
@@ -1,36 +1,36 @@
-import hydrate from 'preact-iso/hydrate';
-import { LocationProvider, Router } from 'preact-iso/router';
-import lazy, { ErrorBoundary } from 'preact-iso/lazy';
+import { LocationProvider, Router, lazy, ErrorBoundary, hydrate } from 'preact-iso';
 
 import pages from 'dir:./pages';
 
 // Generate a Route component and URL for each "page" module:
 const routes = pages.map(name => ({
-  Route: lazy(() => import(`./pages/${name}`)),
-  url: '/' + name.replace(/(index)?\.\w+$/, '')  // strip file extension and "index"
+	Route: lazy(() => import(`./pages/${name}`)),
+	url: '/' + name.replace(/(index)?\.\w+$/, '') // strip file extension and "index"
 }));
 
 function App() {
-  return (
-    <LocationProvider>
-      <div class="app">
-        <nav>
-          <a href="/">Home</a>
-          <a href="/about">About</a>
-        </nav>
-        <ErrorBoundary>
-          <Router>
-            {routes.map(({ Route, url }) => <Route path={url} />)}
-          </Router>
-        </ErrorBoundary>
-      </div>
-    </LocationProvider>
-  );
+	return (
+		<LocationProvider>
+			<div class="app">
+				<nav>
+					<a href="/">Home</a>
+					<a href="/about">About</a>
+				</nav>
+				<ErrorBoundary>
```

Comment:
Might be good to remove `ErrorBoundary` here too - it's no longer needed since the Router handles suspensions itself now.

Reply:
Let's do that separately as it's unrelated to moving imports around.